content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -1168,6 +1168,13 @@ Any
 Most recent
 : Return data at the most recent resource version. The returned data must be
   consistent (in detail: served from etcd via a quorum read).
+  In newer Kubernetes versions (v1.28+),
+  a feature called "consistent reads from cache" is introduced (enabled by default in v1.31+),
+  requiring a newer etcd version (v3.4.33+ or v3.5.13+).
+  This allows the API server to serve some consistent reads directly from its watch cache,
+  a faster in-memory store that mirrors the state of etcd,
+  significantly improving the performance of list requests while maintaining
+  consistency guarantees when the cache is sufficiently up-to-date.
```

Comment:
```suggestion
  Kubernetes {{< skew currentVersion >}} serves “most recent” reads from the _watch cache_:
  an internal, in-memory store within the API server that caches and mirrors the state of data
  persisted into etcd. Kubernetes uses etcd features to maintain cache consistency against 
  the etcd persistence layer. Kubernetes versions v1.28 through to v1.30 also supported this
  feature, although it was not enabled by default until the v1.31 release.
```

I've framed this to be [timeless](https://developers.google.com/style/timeless-documentation); if we leave this in beta for the next 5 releases, the docs stay correct.

Reply:
Rewrote it a little, please take another look.